### ISSUE TITLE
Naming clash due to duplicate REGULAR objects in swan.components inpgrid and cgrid

### ISSUE THREAD

**Issue by pbranson** [rom-py/rompy-legacy#114]
When working with Alson on a SWAN model setup we started getting validation errors on our previously working CGRID because we had combined imports to the top of the module/notebook so had inadvertently ended up with:

```
from rompy.swan.components.cgrid import REGULAR
.
.
.
from rompy.swan.components.inpgrid import REGULAR
```

Propose that we should maybe rename them to INPREGULAR and CGRIDREGULAR given that their definition string has different parameters in SWAN also. Thoughts @rafa-guedes?

**Comment by pbranson** [comment]
cc @alsonathif

**Comment by rafa-guedes** [comment]
@pbranson yes I think you are right - It is possible to `import ... as` but I agree we could avoid this namespace issue altogether by renaming them as you suggest. I can look into it

**Comment by pbranson** [comment]
Thanks @rafa-guedes, we added the as to work around it also, also narrowing the scope of imports is also option.
